### PR TITLE
fix: Update README with current OpenSauced logo

### LINKED ISSUE #3250: Replace Readme logo
The logo in this repo's README is the old one, we should replace it with the new one.

image

Replacement:

https://github.com/open-sauced/assets/blob/main/logos/logo-on-dark.png

### PR DESCRIPTION
**Pull request by Lalithkumarponnambalam**
## Related Tickets & Documents
Fixes #3250

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [x] Tier 4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <br>
-  <img alt="OpenSauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
+  <img alt="OpenSauced" src="https://github.com/open-sauced/assets/blob/main/logos/logo-on-dark.png" width="300px">
```
Comment (bdougie) [inline README.md:3]: Seeing this live. The image seems small. can you swap it with this one
```suggestion
  <img alt="OpenSauced" src="https://github.com/open-sauced/assets/blob/main/logo.png?raw=true" width="300px">
```
Screenshot 2024-04-25 at 11 43 58 AM

**Review by bdougie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bdougie) [reply README.md:3]: oh nvm, I see the width is 300px. can we remove that to make it bigger?

Comment (Lalithkumarponnambalam) [reply README.md]: I will remove the width of 300px, @bdougie. Can you check if the picture looks good now ?

**Review by bdougie — APPROVED** [review]
Thanks @Lalithkumarponnambalam

**Review by nickytonline — APPROVED** [review]
Thanks for the logo update @Lalithkumarponnambalam!

:shipit:

**Pull request merged**